packages/codemods/src/v5/keep-previous-data/__testfixtures__/default.input.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import axios from 'axios'
+import { useQueries, useQuery } from '@tanstack/react-query'
+
+type Post = {
+  id: number
+  title: string
+  body: string
+}
+
+export const Example1 = () => {
+  const queryFn = async (): Promise<Array<Post>> => {
+    const { data } = await axios.get(
+      'https://jsonplaceholder.typicode.com/posts',
+    )
+    return data
+  }
+  const { data } = useQuery({
+    queryKey: ['posts'],
+    queryFn: queryFn,
+    keepPreviousData: true,
+  })
+
+  return <div>{JSON.stringify(data)}</div>
+}
+
+export const Example2 = () => {
+  const queryFn = async (): Promise<Array<Post>> => {
+    const { data } = await axios.get(
+        'https://jsonplaceholder.typicode.com/posts',
+    )
+    return data
+  }
+  const { data } = useQueries({
+    queryKey: ['posts'],
+    queryFn: queryFn,
+    keepPreviousData: true,
+    placeholderData: (previousData) => previousData
```

Comment:
in v4, `placeholderData` didn't have a param, so I'd change this example to:

```
```suggestion
    keepPreviousData: true,
    placeholderData: () => "somePlaceholderData"
```

also, can we add an example where placeholderData isn't a function:

```
keepPreviousData: true,
placeholderData: "somePlaceholderData"
```

and then maybe some combos where we only have `placeholderData`, but not `keepPreviousData`, which shouldn't be touched at all.

Reply:
- a0b452d47d9b8228b824f46b0a2196200bf943fa
- 1813e22a3c84cf4016072c6691ef05e5b35d8e46